### ISSUE TITLE
when restarting from intel gdm shows blinking cursor

### ISSUE THREAD

**Issue by Fludixx** [Askannz/optimus-manager#172]
If i restart my laptop in nvidia mode (intel is the startup mode) then gdm works fine, but if i restart it in intel mode (again, intel is the startup mode) gdm does not work correctly and all i see is a blinking cursor. If i use sddm those problems don't occur

I am on arch with `optimus-manager 1.2.2-1` & `gdm-prime 3.34.1-1`

My configuration:
```
[intel]
DRI=3
accel=
driver=modesetting
modeset=no
tearfree=

[nvidia]
DPI=96
PAT=yes
modeset=no
options=overclocking

[optimus]
auto_logout=yes
pci_power_control=no
pci_remove=no
pci_reset=no
switching=bbswitch
```

output of `journalctl -u gdm.service -b0` when switching in an tty:
```
-- Logs begin at Fri 2019-09-27 20:43:50 CEST, end at Thu 2019-10-31 05:30:33 CET. --
Okt 31 05:30:03 archbook systemd[1]: Starting GNOME Display Manager...
Okt 31 05:30:03 archbook systemd[1]: Started GNOME Display Manager.
```

**Comment by alexandru-calinoiu** [comment]
This happens to me also, I use [this faq article](https://github.com/Askannz/optimus-manager/wiki/FAQ,-common-issues,-troubleshooting#after-trying-to-switch-gpus-i-am-stuck-with-a-black-screen-or-a-black-screen-with-a-blinking-cursor-or-a-tty-login-screen) to solve the issue

**Comment by gabrielsimoes** [comment]
Did you find a solution for it? I have also been unable to solve it, despite trying the settings suggested in the FAQ.

**Comment by Fludixx** [comment]
@gabrielsimoes  I have not found a solution. I just use sddm in the meantime

**Comment by Askannz** [comment]
Late reply, but please provide all the logs from `/var/log/optimus-manager/`. Also try with all power management options disabled (set `switching=none` and in your config above).

**Comment by Fludixx** [comment]
Thank you for your reply. The problem still occurs after setting switching to none.

boot_setup.log: https://pastebin.com/WXXcd7uu
gpu_setup.log: https://pastebin.com/LWUpJbYU
prime_setup.log: https://pastebin.com/pNhyJVGS

**Comment by Fludixx** [comment]
i don't know what fixed it, but after the upgrade to gnome 3.36 everything works fine
